test/bdd/features/search.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Searching for assets on the DKG
+
+    Scenario: Create an asset with keyword
+        Given I setup 4 nodes in my local env
+        When I create an asset with keyword
```

Comment:
For this test you can use already defined steps (look at release.feature)
When I call publish on node 1 with validAssertion with keywords:
    | keyword 1 | keyword 2 |
    And I wait for last publish to finalize
    And Last publish finished with status: COMPLETED